src/fragment-components/a-loading.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import {
+	Checkbox,
+	Loading,
+	TextInput
+} from 'carbon-components-react';
+import { AComponent } from './a-component';
+import { ComponentInfo } from '.';
+import { css } from 'emotion';
+import image from './../assets/component-icons/loading.svg';
+import {
+	nameStringToVariableString,
+	reactClassNamesFromComponentObj,
+	angularClassNamesFromComponentObj
+} from '../utils/fragment-tools';
+
+
+const overlayStyle = css`
+.bx--loading-overlay {
+	position: absolute;
+};`;
+
+export const ALoadingSettingsUI = ({ selectedComponent, setComponent }: any) => {
+	return <>
+		<Checkbox
+			labelText='Small'
+			id='small'
+			checked={selectedComponent.small}
```

Comment:
`size` might make more sense

Reply:
Checkbox is still fine if that's what users expect but it could change the size property and be checked based on a certain value of size.